pkg/webui/console/views/overview/index.js

```diff
@@ -247,6 +248,23 @@ export default class Overview extends React.Component {
                   component="h2"
                 />
               )}
+              <Message
+                className={style.getStarted}
+                content={m.needHelp}
+                values={{
+                  documentationLink: (
+                    <Link.Anchor secondary href={documentationBaseUrl}>
+                      <Message content={sharedMessages.documentation} />
+                    </Link.Anchor>
+                  ),
+                  supportLink: (
+                    <Link.Anchor secondary href={supportLink}>
```

Comment:
```suggestion
                    <Link.Anchor secondary href={supportLink} target="_blank">
```

@laurensslats Do we really need the support link in ttn?

Reply:
Please also check if `supportLink` exists similar to how its done is footer. Currently, with default configuration `supportLink` is empty.

Reply:
I implemented it the following way: If no `supportLink` is going to be added in the configuration, then `need help `text will change (see last screenshot)

Reply:
@bafonins, I've talked to @laurensslats, support links should be the following:
ttn: https://www.thethingsnetwork.org/support
ttes: thethingsindustries.com/support